2000_Furusawa/src/template/mat.dat

```diff
@@ -1,33 +1,24 @@
 MEDIUM 1
```

Comment:
You shoudn't remove the Czub directory with this PR... Can you add the files back?